src/main/java/seedu/address/logic/parser/PomCommandParser.java

```diff
@@ -14,10 +14,6 @@ public PomCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_TIMER);
 
```

Comment:
Maybe add the message that time must be positive

Reply:
I put that under the MESSAGE_USAGE in PomCommand hahaha. In a general invalid response.